Cargo.toml

```diff
@@ -34,6 +34,7 @@ reqwest = { version = "0.10", optional = true }
 serde = { version = "1", features = ["derive"] }
 serde_json = "1"
 surf = { version = "2.0", optional = true }
+once_cell = "1.5.2"
```

Comment:
Let's make this version constraint a bit looser. I think just "1" should be fine here.